### ISSUE TITLE
Static website commands are not supported for attached accounts.

### ISSUE THREAD

**Issue by sofiaaaaaa** [microsoft/vscode-azurestorage#889]
Does this occur consistently? 
Repro steps:

1.
2.

Action: azureStorage.deployStaticWebsite
Error type: Error
Error Message: Static website commands are not supported for attached accounts.

Version: 0.11.0
OS: darwin
OS Release: 20.1.0
Product: Visual Studio Code
Product Version: 1.51.1
Language: en

<details>
<summary>Call Stack</summary>

```
Object.<anonymous> extension.bundle.js:310:107601
extension.bundle.js:310:107226
n extension.bundle.js:310:106912
Object.selectStorageAccountTreeItemForCommand extension.bundle.js:310:107450
extension.bundle.js:515:401583
fulfilled extension.bundle.js:515:399888
processTicksAndRejections task_queues.js:94:5
```

</details>

**Comment by nturinski** [comment]
Could you provide detailed repro steps, please?

**Comment by sofiaaaaaa** [comment]
![image](https://user-images.githubusercontent.com/36926568/100948091-6e575000-354a-11eb-942d-72b5271890ad.png)
Hi, I tried to deploy from attached storage accounts > My stoarge accunt > blob containers > $web, but it didn't allow my deply request, but finally I found the solution that was the way using login my azure account and drop down my storages from azure account "Key" shaped icon.

Thank you. It has solved.
